addon/serializers/json-api-serializer.js

```diff
@@ -214,8 +214,15 @@ const JSONAPISerializer = Serializer.extend({
 
   typeKeyForModel(model) {
     return dasherize(pluralize(model.modelName));
-  }
+  },
 
+  getCoalescedIds(request) {
+    let ids = request.queryParams && request.queryParams['filter[id]'];
+    if (typeof ids === 'string') {
+      return ids.split(',').map((id) => parseInt(id, 10));
```

Comment:
We shouldn't assume integer IDs

Reply:
@lukemelia Agree, good catch! Removed that!